asciidoctor.gemspec

```diff
@@ -1,5 +1,10 @@
 # encoding: UTF-8
-require File.expand_path '../lib/asciidoctor/version', __FILE__
+begin
+  require 'asciidoctor/version'
+rescue LoadError
+  require_relative 'lib/asciidoctor/version'
```

Comment:
We still need to use `require File.expand_path '../lib/asciidoctor/version', __FILE__` to keep the 1.8.7 build running (though we'll be dropping that support shortly).

Reply:
Ok, cool. I amended the commit. Thanks! :)